locking/gradle.lockfile

```diff
@@ -0,0 +1,70 @@
+# This is a Gradle generated file for dependency locking.
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2021-22060   
**Check Name:** springframework: Additional Log Injection in Spring Framework (follow-up to CVE-2021-22096)   
**Severity:** MEDIUM   
**Fixed Version:** 5.3.14   
**Description:** In Spring Framework versions 5.3.0 - 5.3.13, 5.2.0 - 5.2.18, and older unsupported versions, it is possible for a user to provide malicious input to cause the insertion of additional log entries. This is a follow-up to CVE-2021-22096 that protects against additional types of input and in more places of the Spring Framework codebase.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2021-22060